overlord/snapstate/snapstate.go

```diff
@@ -471,20 +471,20 @@ func defaultContentPlugProviders(st *state.State, info *snap.Info) []string {
 	return out
 }
 
-func GetFeatureFlagBool(tr *config.Transaction, flag string) (bool, error) {
-	var v interface{} = false
+func GetFeatureFlagBool(tr *config.Transaction, flag string, unset bool) (bool, error) {
```

Comment:
How do you feel about emulating default args like: http://paste.ubuntu.com/p/5rYxwwvx54 ? This way we can write "GetFeatureFlagBool(tr, "something") and only add "GetFeatureFlagBool(tr, "foo", true)" if the default is different?

Reply:
Ah, that's nice. Yeah.